src/common/Command.cpp

```diff
@@ -493,11 +493,13 @@ namespace Cmd {
         return *Cmd::GetEnv();
     }
 
+#ifndef BUILD_ENGINE
```

Comment:
Please leave a comment for this ifdef? Are we basically only allowing this in the dummy app?

Reply:
There is a comment explaining this where the constructor is declared in the headers. It is allowed only for gamelogic.